### PR TITLE
BaSP-TG-106: Employee - Profile Page

### PR DESCRIPTION
**Pull request by CristianBucci**
- Add form to edit employee's data
- Add 'Save changes' data, to update the data
- Add 'Change password' button (no functionality)
- Add 'Delete account' button (no functionality)
- Add Profile image

QA: 
How to test?:
- The profile page must show the employee's info: profile picture, name, email, address and phone. (All data has to be loaded on the inputs)
- The 'Save changes' button must update the data of the employee correctly
- The 'Change password' and 'Delete account' buttons won't have a functionality associated.

### CODE REVIEW

File: src/Components/Employees/Profile/index.js [hunk 1]
```diff
@@ -1,13 +1,170 @@
...
+            name="phone"
+            type="text"
+            error={errors.phone?.message}
+            placeholder={'Phone'}
+          />
+          <div>
+            <Buttons type="button" variant="primary" name="Change password" />
+          </div>
+          <div>
+            <Buttons type="submit" variant="primary" name="Save channges" />
```
Comment (matescarabino) [inline src/Components/Employees/Profile/index.js:152]: Typo
```suggestion
            <Buttons type="submit" variant="primary" name="Save changes" />
```

**Review by matescarabino — CHANGES_REQUESTED** [review]
I would also change the color of the "save changes" button, since it is the main one of this form and it does not stand out. But this is an UX / UI thing

**Review by matescarabino — APPROVED** [review]
LGTM

**Review by Ansitox — APPROVED** [review]
Everything look great, GJ

**Review by MaxiG-dev — APPROVED** [review]
Looks good for me! ✅

**Review by NicoSan02 — APPROVED** [review]

**Pull request merged**